master.js

```diff
@@ -0,0 +1,151 @@
+let data = [];
```

Comment:
I would recommend having all these methods and attributes in a class to manage them well instead of throwing them like this